### DISCUSSION TITLE
p/30b1b683648a4617a5c04bd6e234dddd/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#53] [Announcements]
# p/30b1b683648a4617a5c04bd6e234dddd/

距离上次写周报又过去了两周，这两周青甘线花费了太多的时间，毕竟地处西北偏僻之地，人烟稀少交通不便，所以花费了比较多的时间。  西安去西安看了一下兵马俑和秦始皇陵，前者的规模并不太大，后者看起来风景还不错。另外西安似乎也和河南一样，几乎到处都是汉服和汉服，尤其是夜晚的鼓楼附近，真的超级多。吾辈也在思考，是否因为中原地区从历史上看一直都是动乱的地区，从未流传过下来什么民族民族服装之类的（像是南方的广西

https://blog.rxliuli.com/p/30b1b683648a4617a5c04bd6e234dddd/

**Comment by dohard-ma** [comment]
感谢分享，非常有意义👍🎇

**Comment by rxliuli** [comment]
谢谢，希望帮助到你😄

**Comment by yy16** [comment]
什么家庭啊，到处玩不用上班吗

**Reply by rxliuli** [reply]
上班是不可能上班的，这辈子都不可能上班的🤣
